src/lib/utils/formatCurrency.ts

```diff
@@ -0,0 +1,17 @@
+import { Helpers } from '@ardenthq/sdk-profiles';
+
+const formatCurrency = (
+    value: number,
+    ticker: string,
+    options?: {
+        locale?: string;
+        withTicker?: boolean;
+    },
+) => {
+    return Helpers.Currency.format(value, ticker, {
+        ...options,
+        locale: options?.locale ?? 'en-US',
```

Comment:
Won't this always result in `en-US` being used as nothing passes the locale to this method?